runtime/lua/vim/provider/health.lua

```diff
@@ -426,7 +431,7 @@ end
 --- @param url string
 local function download(url)
   local has_curl = vim.fn.executable('curl') == 1
-  if has_curl and vim.fn.system({ 'curl', '-V' }):find('Protocols:.*https') then
+  if has_curl and cmd_ok({ 'curl', '-V' })[2]:find('Protocols:.*https') then
```

Comment:
this isn't going to work. multi-return can't be indexed. need to either surround with `({ ... })` or save to a local var.